quickstarts-js/Image_out.js

```diff
@@ -0,0 +1,469 @@
+
+![Macaron prepartation step 7](https://storage.googleapis.com/generativeai-downloads/images/macaron_step7.png)
+
+Finally, let them mature in the refrigerator for at least 24 hours. This allows the flavors to meld and the shells to soften to the perfect chewy consistency.
+
+Enjoy your homemade macarons!
+
+-----
+*/
+
+/* Markdown (render)
+## Chat mode (recommended method)
+
+So far you've used unary calls, but Image-out is actually made to work better with chat mode as it's easier to iterate on an image turn after turn.
+*/
+
+// [CODE STARTS]
+chat = ai.chats.create({
+    model: MODEL_ID
+})
+// [CODE ENDS]
+
+// [CODE STARTS]
+message = "create a image of a plastic toy fox figurine in a kid's bedroom, it can have accessories but no weapon";
+
+response = await chat.sendMessage({ message: message });
+
+for (const part of response.candidates[0].content.parts) {
+  if (part.text !== undefined) {
+    console.log(part.text);
+  }
+  if (part.inlineData !== undefined) {
+    foxFigurineImage = part.inlineData.data
+    console.image(foxFigurineImage);
+  }
+}
+
+// [CODE ENDS]
+
+/* Output Sample
+
+Here is an image of a plastic toy fox figurine in a kid&#x27;s bedroom, with accessories: 
+
+<img src="https://iili.io/K2A8OAJ.png" style="height:auto; width:100%;" />
+
+*/
+
+// [CODE STARTS]
+message = "Add a blue planet on the figuring's helmet or hat (add one if needed)";
+
+response = await chat.sendMessage({ message: message });
+
+for (const part of response.candidates[0].content.parts) {
+  if (part.text !== undefined) {
+    console.log(part.text);
+  }
+  if (part.inlineData !== undefined) {
+    console.image(part.inlineData.data);
+  }
+}
+
+// [CODE ENDS]
+
+/* Output Sample
+
+Here&#x27;s the toy fox figurine with a blue planet on its helmet: 
+
+<img src="https://iili.io/K2A8CyG.png" style="height:auto; width:100%;" />
+
+*/
+
+// [CODE STARTS]
+message = "Move that figurine on a beach";
+
+response = await chat.sendMessage({ message: message });
+
+for (const part of response.candidates[0].content.parts) {
+  if (part.text !== undefined) {
+    console.log(part.text);
+  }
+  if (part.inlineData !== undefined) {
+    console.image(part.inlineData.data);
+  }
+}
+
+// [CODE ENDS]
+
+/* Output Sample
+
+Here&#x27;s the figurine on a beach! 
+
+<img src="https://iili.io/K2AvYIR.png" style="height:auto; width:100%;" />
+
+*/
+
+// [CODE STARTS]
+message = "Now it should be base-jumping from a spaceship with a wingsuit";
+
+response = await chat.sendMessage({ message });
+ 
+for (const part of response.candidates[0].content.parts) {
+  if (part.text !== undefined) {
+    console.log(part.text);
+  }
+  if (part.inlineData !== undefined) {
+    console.image(part.inlineData.data);
+  }
+}
+
+// [CODE ENDS]
+

```

Comment:
I think you are missing some of the examples here

Reply:
https://storage.googleapis.com/generativeai-downloads/cookbook/image_out/figurine_bbq.jpg
https://storage.googleapis.com/generativeai-downloads/cookbook/image_out/figurine_spa.jpg